64563 Request - Inactive Flag on Machine File

rahulrawatjkt opened the pull request:
Files:
est/d-estop.w
schedule/objects/resources.w
touch/allmachs.w
touch/machines.w
windows/l-mach.w

rahulrawatjkt committed:
64563 Request - Inactive Flag on Machine File
Files:
est/d-estop.w
schedule/objects/resources.w
touch/allmachs.w
touch/machines.w
windows/l-mach.w

ron-advantzware reviewed (changes requested):
When using a logical field in a WHERE phrase, it is a best practice as recommended by Progress and Dan Forman to not simply use WHERE <LogicalField> without explicitly adding the comparison component.  Please can WHERE <LogicalField> to WHERE <LogicalField> EQ YES and WHERE NOT <LogicalField> to WHERE <LogicalField> EQ NO.  Without the comparison component, Progress does not utilize the <LogicalField> if part of an INDEX and you may inadvertently cause Progress to perform sequential search.

rahulrawatjkt committed:
64563 Request - Inactive Flag on Machine File
FIles:
schedule/objects/resources.w
touch/allmachs.w
touch/machines.w
windows/l-mach.w

rahulrawatjkt committed:
64563 Request - Inactive Flag on Machine File
Files:
schedule/objetcs/resources.w
